fail to compile the code

getting the following syntax error when compiling the code following instructions from here 
https://github.com/Snowflake-Labs/pg_lake/blob/main/docs/building-from-source.md

this step
export VCPKG_VERSION=2025.01.13 && \
git clone --recurse-submodules https://github.com/Microsoft/vcpkg.git && \
./vcpkg/bootstrap-vcpkg.sh && \
./vcpkg/vcpkg install azure-identity-cpp azure-storage-blobs-cpp azure-storage-files-datalake-cpp openssl && \
export VCPKG_TOOLCHAIN_PATH="$(pwd)/vcpkg/scripts/buildsystems/vcpkg.cmake"



ubuntu, gcc  14.2

pg_lake/duckdb_pglake/pg_lake/duckdb_pglake/src/fs/caching_file_system.cpp:203:70: error: expected ')' before 'PRId64'
  203 |                 PGDUCK_SERVER_DEBUG("Total bytes will be written (%" PRId64 ") is greater than "

--- a/duckdb_pglake/src/fs/caching_file_system.cpp
+++ b/duckdb_pglake/src/fs/caching_file_system.cpp
@@ -200,12 +200,16 @@ CachingFileSystem::ShouldCacheOnWrite(CachingFSFileHandle &pg_lakeHandle, int64_
 	int64_t cacheOnWriteMaxAllowedSize = setting.GetValue<uint64_t>();
 	if (pg_lakeHandle.cacheOnWriteWrittenBytes + additionalByteCount >= cacheOnWriteMaxAllowedSize)
 	{
-		PGDUCK_SERVER_DEBUG("Total bytes will be written (%" PRId64 ") is greater than "
-							"pg_lake_cache_on_write_max_size (%" PRId64 "). "
-							"Disabling cache-on-write for this file %s%s.",
-							pg_lakeHandle.cacheOnWriteWrittenBytes + additionalByteCount,
-							cacheOnWriteMaxAllowedSize,
-							pg_lakeHandle.cacheOnWritePath.c_str(), ".pgl-stage");
+		PGDUCK_SERVER_DEBUG(
+			"Total number of bytes that will be written (%" PRId64 ") is greater than "
+			"pg_lake_cache_on_write_max_size (%" PRId64 "). "
+			"Disabling cache-on-write for this file %s%s.",
+			pg_lakeHandle.cacheOnWriteWrittenBytes + additionalByteCount,
+			cacheOnWriteMaxAllowedSize,
+			pg_lakeHandle.cacheOnWritePath.c_str(),
+			".pgl-stage"
+		);
+
 		return false;
 	}
 

